src/node_api_backport.h

```diff
@@ -0,0 +1,53 @@
+#ifndef SRC_NODE_API_BACKPORT_H_
+#define SRC_NODE_API_BACKPORT_H_
+
+#include "node_internals.h"
+#include "env-inl.h"
+
```

Comment:
Can you add some lines of comment in this file, explaining that this is for "faking" async_hooks support as it's needed by napi?